### PR TITLE
Update contributors.json

### PR DESCRIPTION
**Pull request by Swapnil074**

### CODE REVIEW

File: contributors.json [hunk 1]
```diff
@@ -154,5 +154,9 @@
   "WickedInvi": {
     "country": "United Kingdom",
     "name": "Deyan Petrov"
+  },
+  "swpanil074": {
```
Comment (subeshb1) [inline contributors.json:158]: Let's keep this sorted in ascending  order by GitHub user name

**Review by subeshb1 — CHANGES_REQUESTED** [review]
Let's keep `contributors.json` sorted in ascending order by GitHub user name

**Comment by Swapnil074** [comment]
> Let's keep `contributors.json` sorted in ascending order by GitHub user name
just updated in the order

File: contributors.json [hunk 2]
```diff
@@ -154,5 +158,6 @@
   "WickedInvi": {
     "country": "United Kingdom",
     "name": "Deyan Petrov"
-  }
+  },
```
Comment (subeshb1) [inline contributors.json:161]:
```suggestion
  }
```

File: contributors.json [hunk 3]
```diff
@@ -155,4 +159,5 @@
     "country": "United Kingdom",
     "name": "Deyan Petrov"
   }
+
```
Comment (subeshb1) [inline contributors.json:162]:
```suggestion
```

**Review by subeshb1 — APPROVED** [review]

**Pull request merged**